x/vm/types/precompiles.go

```diff
@@ -13,6 +13,7 @@ const (
 	BankPrecompileAddress         = "0x0000000000000000000000000000000000000804"
 	GovPrecompileAddress          = "0x0000000000000000000000000000000000000805"
 	SlashingPrecompileAddress     = "0x0000000000000000000000000000000000000806"
+	ICS02PrecompileAddress        = "0x0000000000000000000000000000000000000808"
```

Comment:
I took the address `0x0000000000000000000000000000000000000808` instead of `0x0000000000000000000000000000000000000807` because when I grepped the repo, I saw that the latter is used somewhere. Not sure exactly where it is used, but I can take the latter if it is free

Reply:
Let's use 807, that's an accidental artifact left over from the callbacks address, which we determined has no use so removed it.